site/docs/v1/tech/email-templates/email-templates.adoc

```diff
@@ -459,7 +489,9 @@ The User object, see the User API for field definitions of a User.
 [field]#verificationId# [type]#[String]#::
 The verification Id intended to be used by the link:/docs/v1/tech/apis/registrations#verify-a-user-registration[Verify a User Registration] API.
 
-
+[field]#verificationOneTimeCode# [type]#[String]#::
+// TODO Add link to additional documentation as we complete
```

Comment:
Did you mean to leave this TODO in?

Reply:
Kinda, was thinking that we would have more links to add.

Reply:
Okay, cool. Wasn't sure what kind of doco would go there.